public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -231,6 +317,45 @@
         return ResponseEntity.ok(resp);
     }
 
+    @Override
+    public ResponseEntity<ConceptAnalysisListResponse> getConceptAnalysisResults(List<String> conceptIds){
+        ConceptAnalysisListResponse resp=new ConceptAnalysisListResponse();
+        List<ConceptAnalysis> conceptAnalysisList=new ArrayList<>();
+        for(String conceptId: conceptIds){
+            ConceptAnalysis conceptAnalysis=new ConceptAnalysis();
+            AchillesAnalysis genderAnalysis=achillesAnalysisDao.findConceptAnalysisResults(conceptId,3101L);
+            AchillesAnalysis ageAnalysis=achillesAnalysisDao.findConceptAnalysisResults(conceptId,3102L);
+
+            conceptAnalysis.setConceptId(conceptId);
+            if(genderAnalysis != null){
+                for(AchillesResult ar:genderAnalysis.getResults()){
+                    String stratum5_name=ar.getStratum5Name();
```

Comment:
This is my fault i know. I started this stratum5_name transient column stratum5Name ... what gets put here is the gender or age decile name that matches the stratum. But in some analysis that is stratum2 and some it's stratum5 ... Can we rename this on AchillesResults model and everywhere  and call it something common  like ...(looking in thesaurus)  -- probeName ? analysisStratumName  -- analysisStratumName is the best i can come up with.